src/Components/ItemPropertiesV1.jsx

```diff
@@ -0,0 +1,165 @@
+import TreeItem from '@mui/lab/TreeItem';
```

Comment:
I'm generally not a fan of versioned filenames.  Why need this?

Reply:
removed